From 92e66b6caf93e001f6c0045d729359a2fbd6b130 Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Wed, 4 Mar 2009 19:14:54 +0800
Subject: [PATCH] ENGR00108904 MX35: Fix gps failure caused by one missed input iomux

MUX_IN_GPIO2_IN_1 is missed in iomux_input_select
so the later INPUT IOMUX definitions are wrong

Signed-off-by: Wallace Wang <r59996@freescale.com>
---
 arch/arm/mach-mx35/iomux.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx35/iomux.h b/arch/arm/mach-mx35/iomux.h
index c03dfe4..258165c 100644
--- a/arch/arm/mach-mx35/iomux.h
+++ b/arch/arm/mach-mx35/iomux.h
@@ -1,5 +1,5 @@
 /*
- * Copyright 2008 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2008-2009 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -138,6 +138,7 @@ typedef enum iomux_input_select {
 	MUX_IN_GPIO2_IN_17,
 	MUX_IN_GPIO2_IN_18,
 	MUX_IN_GPIO2_IN_19,
+	MUX_IN_GPIO2_IN_1,
 	MUX_IN_GPIO2_IN_20,
 	MUX_IN_GPIO2_IN_21,
 	MUX_IN_GPIO2_IN_22,
-- 
1.5.4.4

